plugins/tgis_grpc_plugin.py

```diff
@@ -61,15 +61,22 @@ def make_request(self, query: dict, user_id: int):
             params=generation_pb2_grpc.generation__pb2.Parameters(
                 method=generation_pb2_grpc.generation__pb2.GREEDY,
                 stopping=generation_pb2_grpc.generation__pb2.StoppingCriteria(
-                    max_new_tokens=query["max_new_tokens"],
-                    min_new_tokens=query["min_new_tokens"],
+                    max_new_tokens=query["output_tokens"],
+                    min_new_tokens=query["output_tokens"],
```

Comment:
Hmm, so we are forcing the model to respond with an exact number of tokens. Probably good for the benchmarking exercises I guess.

Reply:
This is actually how it was before, but in the dataset we had two fields per element (min_new_tokens, max_new_tokens) but they were always the same